src/SQLStore/QueryEngine/Interpreter/ValueDescriptionInterpreter.php

```diff
@@ -87,7 +87,11 @@ public function interpretDescription( Description $description ) {
 				$value
 			);
 
-			$query->where = "{$query->alias}.smw_sortkey$comparator" . $this->querySegmentListBuilder->getStore()->getConnection( 'mw.db' )->addQuotes( $value );
+			$db = $this->querySegmentListBuilder->getStore()->getConnection( 'mw.db' );
+
+			$query->where = "{$query->alias}.smw_sortkey$comparator" . $db->addQuotes( $value ).
+			" AND $query->alias.smw_iw!=" . $db->addQuotes( SMW_SQL3_SMWIW_OUTDATED ) .
```

Comment:
I'd put the concat symbol on the next line and use one more indent before it. This way its a lot easier to see at a glance what is happening